src/map/index.ts

```diff
@@ -6,13 +6,14 @@ import {
   MAPBOX_DARK_STYLE_ID,
   MAPBOX_TOKEN,
 } from './config'
+import { getThemeId } from '~/theme'
 
 export type Coords = [number, number][]
 
 const POLYLINE_SAMPLE_SIZE = 50
 const POLYLINE_PRECISION = 4
 
-function getMapStyleId(themeId: string): string {
+export function getMapStyleId(themeId: string): string {
```

Comment:
I don't think this needs to be exported anymore

Reply:
yep, removed